### PR TITLE
Adds SARJ-LLC xPath scraper

### PR DESCRIPTION
**Pull request by halorrr**

### CODE REVIEW

File: SCENE-SCRAPABLE.md [hunk 1]
```diff
@@ -287,9 +295,12 @@ recklessinmiami.com|RealityKings.yml
 rk.com|RealityKings.yml
 roccosiffredi.com|GammaEntertainment.yml
 roundandbrown.com|RealityKings.yml
+<<<<<<< HEAD
```
Comment (bnkai) [inline SCENE-SCRAPABLE.md:298]: leftover , needs to be removed

**Comment by halorrr** [comment]
Updated

File: scrapers/MetArtNetwork.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="pull-left custom-photo-movie-description"]
+      $credits: //div[@class="pull-left custom-credits-left"]
+      $script: //script[contains(text(),'INITIAL_STATE')]/text()
+    scene:
+      Title: $info/h3/a/@title
+      Details: $info//p/text()
```
Comment (bnkai) [inline scrapers/MetArtNetwork.yml:15]: The details gets more results than it should and it only keeps the short description not the long one.
`https://www.metartnetwork.com/model/mary-moody-and-naomi-swann/movie/20200619/NO_REGRETS/`
`https://www.metartnetwork.com/model/lil-karla/movie/20200618/FORCED_ORGASM_2/`
The below works as it should at least for the above scenes
```yamlDetails: 
        selector: //div[@class="custom-description-long custom-description-hide"]/p/text()
        concat: "\n"
        replace:
          - regex: "\n"
            with: "\n" 
```
the extra replace is used to protect the "\n" as its usage is still troublesome

File: scrapers/MetArtNetwork.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      Date:
+        selector: $credits//li[span/text()='Released:']/text()
+        parseDate: Jan 2, 2006
+      Performers:
+        Name: $credits//li[span/text()='Cast:']//a/text()
+      Tags:
+        Name: //div[@class="tag-container"]//a/text()
+      Image: //div[@class="custom-movie-preview"]//img/@src
+      Studio:
+        Name: //meta[@property="og:site_name"]/@content
```
Comment (bnkai) [inline scrapers/MetArtNetwork.yml:25]: For the studio i couldn't find a scene that matches but neither did i find another way as it doesn't seem to be present in the page. Searching for the title also has too many results to bother with (common titles) .

**Comment by halorrr** [comment]
Updated! Looks like the image selector I had before didn't work as well. I fixed the image and at the same time found a was to fix the studio.

File: scrapers/SARJ-LLC.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
+name: "SARJ LLC"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - metart.com
```
Comment (bnkai) [inline scrapers/SARJ-LLC.yml:5]: metart seems to have changed the layout.

File: scrapers/SARJ-LLC.yml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="info-container"]
+      $script: //script[contains(text(),'INITIAL_STATE')]/text()
+    scene:
+      Title: $info//h3/a/text()
+      Details:
+        selector: $info//p[@class="description-text"]/text()
```
Comment (bnkai) [inline scrapers/SARJ-LLC.yml:25]: in `https://www.sexart.com/model/nancy-a-and-eric-el-gran-and-melody-petite-and-ariel-piper-fawn-and-nick-ross-and-alexis-crystal/movie/20200621/SEXART_HOLIDAY_ON_MYKONOS_EPISODE_6` or `https://www.sexart.com/model/sylvan-and-julia-roca-and-carolina-abril/movie/20200506/THREE_IN_A_BED` or `https://www.thelifeerotic.com/model/lil-karla/movie/20200618/FORCED_ORGASM_2` you can see that you get the short description not the normal.
The normal one at least for sexart, tle is in the script you have in common i think (that you dont seem to use anywhere)

File: SCENE-SCRAPABLE.md [hunk 6]
```diff
@@ -387,7 +399,8 @@ unlimitedmilfs.com|NewSensationsNetworkSites.yml
 vcaxxx.com|Hustler.yml
 vipissy.com|Vipissy.yml
 vipsexvault.com|LetsDoeIt.yml
-vivid.com|Vivid.yml
+vivid.com|GammaEntertainment.ymlvideos
```
Comment (bnkai) [inline SCENE-SCRAPABLE.md:402]: this should be switched back to
`vivid.com|Vivid.yml`

File: scrapers/SARJ-LLC.yml (hunk 4 above)
Comment (halorrr) [reply scrapers/SARJ-LLC.yml:5]: Metart seems to be still the same and working for me? Scene I just tested with https://www.metart.com/model/monika-dee/movie/20200623/ANCHORED

File: scrapers/SARJ-LLC.yml (hunk 5 above)
Comment (halorrr) [reply scrapers/SARJ-LLC.yml]: Updated to use the script instead, tested and it seems to work on all the sites that have descriptions to scrape.

File: scrapers/SARJ-LLC.yml (hunk 4 above)
Comment (bnkai) [reply scrapers/SARJ-LLC.yml:5]: i am getting this response https://pastebin.com/X8szvsui
I think some sort of cdn is interfering
The other sites don't do that though

File: scrapers/MetArtNetwork.yml [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - metartnetwork.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="pull-left custom-photo-movie-description"]
+      $credits: //div[@class="pull-left custom-credits-left"]
```
Comment (bnkai) [inline scrapers/MetArtNetwork.yml:11]: in some scenes the credits tab is missing so we miss the performers and the date
the same with info and title
sample urls 
`https://www.metartnetwork.com/model/monika-dee/movie/20200623/ANCHORED/`
`https://www.metartnetwork.com/model/zelda-b/movie/20200607/SEXY_LUMBERJACK/`

changing the fields to the following seems to work 

```yaml
Title: //h3/a/@title
```
```yaml
      Date:
        selector: //li[@class="navbar-custom-login"]/a/@href
        replace: 
          - regex: .+galleryDate=(\d{8})&.+
            with: $1
        parseDate: "20060102"
```
```yaml
      Performers:
        Name: //a/span[contains(.,"Cast:")]/../text()
```

**Comment by halorrr** [comment]
I've converted this PR to a draft, it has a ton of work it needs to make it work again, I'll return to it later.

**Comment by halorrr** [comment]
@bnkai I have finally updated this scraper. Looks like they finished their site updates as all the sites use the same formatting again so this is once again a single scraper.

**Comment by bnkai** [comment]
@halorrr will test site by site and leave feedback (if needed) when i am done, it might take a couple of days

**Comment by bnkai** [comment]
Works ok , merging.
It also seems to work with galleries ( needs a tuning for the description regex ) , will leave that for a later PR.

**Pull request merged**